metricbeat/module/ceph/cluster_health/cluster_health_integration_test.go

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-// +build integration,linux
+// build integration,linux
```

Comment:
Typo?

Reply:
Yes, sorry about that. Trying some things on Mac (still work-in-progress).